### DISCUSSION TITLE
Welcome to SeaORM

### DISCUSSION THREAD

**Discussion by tyt2y3** [SeaQL/seaql.github.io#3] [General]
https://www.sea-ql.org/SeaORM/blog/welcome

**Comment by exzachlyvv** [comment]
This is exactly what I have been looking for! Thank you so much for all the hard work on SeaQL and SeaORM!

**Reply by billy1624** [reply]
Hi @exzachlyvv, welcome!! :P

**Reply by tyt2y3** [reply]
Thank you and welcome. Definitely share your experience! We are still in early stage and constantly improving so your feedback is definitely appreciated.
